txpipe/sysmaps.py

```diff
@@ -167,6 +173,10 @@ def run(self):
         map_pix, ngals, g1, g2, var_g1, var_g2 = mapper.finalize(self.comm)
         map_pix_psf, ngals_psf, g1_psf, g2_psf, var_g1_psf, var_g2_psf = mapper_psf.finalize(self.comm)
         flag_pixs, flag_maps = flag_mapper.finalize(self.comm)
+        import treecorr
+        cat = treecorr.Catalog(ra = ra, dec = dec, ra_units='degree', dec_units = 'degree', npatch=self.config['npatch'])
+        cat.patches
```

Comment:
This line isn't actually necessary.  (Or shouldn't be anyway).  Calling `write_patch_centers` should be enough to trigger the patch creation.

Reply:
Okay, wasn't sure